src/helpers/replaceDomain.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import parse from 'url-parse';
+
+const replaceDomain = (URL: string, domain: string) => {
+  const parsed = parse(URL);
```

Comment:
what is we `const parsed = new URL(myUrlString)`? We'd save the dependency on `url-parse`.